### PR TITLE
Layout and nav updates

### PR DESCRIPTION
**Pull request by ChristianPavilonis**
- **adjusts alignment for main content**
- **Spaces footer away from content darkens the border color**

### CODE REVIEW

**Comment by ChristianPavilonis** [comment]
Maybe we should include a CTA in the Nav for submitting an article.

It would just link to the readme I guess.

**Comment by IroncladDev** [comment]
You are allowed to be absolutely unhinged with your commit messages, btw

File: app/layout.tsx [hunk 1]
```diff
@@ -22,8 +26,69 @@ export default function RootLayout({
       <body
         className={`${jetBrainsMono.className} antialiased bg-base text-text`}
       >
+        <header className="max-w-[1280px] w-full mx-auto px-4 pt-8 flex items-center justify-between pb-8">
```
Comment (IroncladDev) [inline app/layout.tsx:29]: Try to use the `styled()` util from `react-tailwind-variants` for long classes like this

```tsx
const Header = styled("header", {
  base: "max-w-[1280px] w-full mx-auto px-4 pt-8 flex items-center justify-between pb-8"
});
```

You can also do

```tsx
<Flex className="max-w-[1280px] !pl-0" p={8} center width="full" asChild>
  <header>...</header>
</Flex>
```

File: tailwind.config.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ export default {
       colors: {
         background: "var(--background)",
         foreground: "var(--foreground)",
+        primary: "#ba43f7"
```
Comment (IroncladDev) [inline tailwind.config.ts:15]: Let's use the `mauve` color in the [catppuccin](https://github.com/catppuccin/catppuccin) palette

**Review by IroncladDev — CHANGES_REQUESTED** [review]
Some changes

I apologize at how nit-picky I am about clean JSX

File: app/components/logo-mark.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+export default function LogoMark(props: {className?: string}) {
```
[the remark this answers is not in the document]
Comment (ChristianPavilonis) [reply app/components/logo-mark.tsx]: I did this, but I needed a [build dep](https://github.com/TehcCringe/TehcCringe/pull/43/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R33) to inline the svg, lmk if that's what you prefer.

img tags don't let us style outside of the svg file itself, so we wouldn't be able to modify the fill color on hover for example.

**Review by IroncladDev — APPROVED** [review]
LGTM

**Pull request merged**